src/dashboard/src/components/administration/models.py

```diff
@@ -50,3 +51,4 @@ class ArchivesSpaceConfig(models.Model):
     access_conditions = models.CharField(max_length=50, blank=True, verbose_name='Conditions governing access')
     use_conditions = models.CharField(max_length=50, blank=True, verbose_name='Conditions governing use')
     repository = models.IntegerField(default=2, verbose_name='ArchivesSpace repository number')
+    inherit_notes = models.BooleanField(default=True, verbose_name='Inherit digital object notes from the parent component')
```

Comment:
Since the inherit_notes is particular for a client, do we want to set the default to True?